3.9 Afternoon - Android Agent 6.10.0 release notes & doc updates (#11428)
newrelic/docs-website: src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx

---
title: "shutdown (Android SDK API)"
type: apiDoc
shortDescription: Shut down the agent during runtime
tags:
  - Mobile monitoring
  - New Relic Mobile Android
  - Android SDK API
metaDescription: 'New Relic API for Android mobile app monitoring: shut down agent.'
redirects:
  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down
---

## Syntax

```
NewRelic.shutdown()
```

Shut down the agent within the current application lifecycle during runtime

## Requirements

Agent version 6.10.0 or higher.

## Description

This API provides the ability to shutdown the agent within the current application lifecycle during runtime. Once the agent has shutdown, it cannot be restarted within the current application lifecycle.

## Examples

> ### Shut down agent

Reviewer: ```suggestion
### Shut down the Android agent
```

> Example of shutting down the agent:

Reviewer: ```suggestion
Here's an example of shutting down the agent:
```

```
NewRelic.shutdown()

> ```

Reviewer: @ywang-nr Can we build this example out more? Are there any scenarios where something would go in the brackets () of NewRelic.shutdown()?

Author: This is all the developers need for shutting down the agent, no params are needed

---
title: "shutdown (Android SDK API)"
type: apiDoc
shortDescription: Shut down the agent during runtime
tags:
  - Mobile monitoring
  - New Relic Mobile Android
  - Android SDK API
metaDescription: 'New Relic API for Android mobile app monitoring: shut down agent.'
redirects:
  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down
---

## Syntax

```
NewRelic.shutdown()
```

> Shut down the agent within the current application lifecycle during runtime

Reviewer: ```suggestion
Shut down the agent within the current application lifecycle during runtime.
```

## Requirements

Agent version 6.10.0 or higher.

## Description

> This API provides the ability to shutdown the agent within the current application lifecycle during runtime. Once the agent has shutdown, it cannot be restarted within the current application lifecycle.

Reviewer: ```suggestion
This API provides the ability to shut down the agent within the current application lifecycle during runtime. Once the agent has shut down, it cannot be restarted within the current application lifecycle.
```

Reviewer: @ywang-nr What's a scenario where the user would want to use this API?

Author: This feature is meant to give the developer options to disable our agent, most likely related to their end users' privacy issue

> For general info on using this API, see the [Android SDK API usage guide](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/work-android-sdk-api).

Reviewer: ```suggestion
```
